api/v1/postgres_types.go

```diff
@@ -205,6 +215,54 @@ func (p *Postgres) ToKey() *types.NamespacedName {
 	}
 }
 
+func (p *Postgres) ToSvcLB(lbIP string, lbPort int32) *corev1.Service {
+	lb := &corev1.Service{}
+	lb.Spec.Type = "LoadBalancer"
+
+	lb.Annotations = map[string]string{
+		"metallb.universe.tf/allow-shared-ip": "spilo",
+	}
+
+	lb.Namespace = p.ToPeripheralResourceNamespace()
+	lb.Name = p.ToSvcLBName()
+	label := map[string]string{
+		"application":  "spilo",
+		"cluster-name": p.ToPeripheralResourceName(),
+		"spilo-role":   "master",
+		"team":         p.generateTeamID(),
+	}
+	lb.SetLabels(label)
+
+	// svc.Spec.LoadBalancerSourceRanges // todo: Do we need to set this?
+
+	port := corev1.ServicePort{}
+	port.Name = "postgresql"
+	port.Port = lbPort
+	port.Protocol = corev1.ProtocolTCP
+	port.TargetPort = intstr.FromInt(5432)
+	lb.Spec.Ports = []corev1.ServicePort{port}
+
+	lb.Spec.Selector = label
```

Comment:
Make sure *not* to use our custom label here, as this is the selector for operator-managed pods